src/comics/chapters/routes.rs

```diff
@@ -62,6 +63,10 @@ pub fn chapters_router() -> Router<AppState> {
         .route("/:comic_id/chapters", get(get_chapters))
         .route("/chapters/:chapter_id", delete(delete_chapter))
         .route("/chapters/:chapter_id", put(update_chapter))
+        .route(
+            "/chapters/by_slug/:username/:slug/:chapter_number/",
```

Comment:
I think it's better if it's like `/api/v1/comics/by_slug/:comic_slug/chapters/:chapter_number` no?

I think this is more consistent with the rest conventions and also makes it less prone to overlapping stuff

Reply:
What about `:username`?

Reply:
yeah, idk, we have 3 options, either stick it at the end like `/api/v1/comics/by_slug/:comic_slug/chapters/:chapter_number/:username` or `/api/v1/comics/by_slug/:comic_slug/chapters/:chapter_number/users/:username` or `/api/v1/users/:username/comics/by_slug/:comic_slug/chapters/:chapter_number`